### PR TITLE
Add all collections page

### PR DESCRIPTION
**Pull request by mariano-aguero**
Connects #322.
Connects #324.

### Includes:
Based on branch `sort-filter-collectibles-list-url`, the last two commits are the importants
Page according to this design
Screenshot_20190522_162304

### Does not include
- Most traded row

### CODE REVIEW

**Comment by unjapones** [comment]
The `blocked by` label will remain in-place because #416 (just added to the description) adds the "Sell Collectible" button on the top-right corner.

I believe this PR can still be styled though.

File: src/components/erc721/erc721_app.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ export const Erc721App = () => (
         <Switch>
             <Route exact={true} path={`${ERC721_APP_BASE_PATH}/`} component={AllCollectibles} />
             <Route exact={true} path={`${ERC721_APP_BASE_PATH}/my-collectibles`} component={MyCollectibles} />
+            <Route exact={true} path={`${ERC721_APP_BASE_PATH}/list-collectibles`} component={ListCollectibles} />
```
Comment (fvictorio) [inline src/components/erc721/erc721_app.tsx:26]: PP: `/collectibles`. (Note that if you change this you also have to change the links).

**Comment by gabitoesmiapodo** [comment]
This should look more or less like this now.

Screen Shot 2019-05-23 at 11 43 25
Screen Shot 2019-05-23 at 11 44 38
Screen Shot 2019-05-23 at 11 44 51

File: src/components/erc721/collectibles/collectibles_all.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                            text="View all"
+                            to={`${ERC721_APP_BASE_PATH}/list-collectibles?filter=${
+                                CollectibleFilterType.ShowAll
+                            }&sort=${CollectibleSortType.NewestAdded}`}
+                        />
+                    </SubSectionTitleWrapper>
+                    <CollectiblesCardListStyled
+                        collectibles={collectibles}
+                        filterType={CollectibleFilterType.ShowAll}
+                        limit={5}
```
Comment (Agupane) [inline src/components/erc721/collectibles/collectibles_all.tsx:130]: PP: This should be a constant

**Review by Agupane — APPROVED** [review]

**Pull request merged**